### PR TITLE
Colors for light and dark theme on CSS custom prop declaration and in…

### PR DESCRIPTION
**Pull request by chiangs**
## Dark
Dark

## Light
Light

### CODE REVIEW

File: themes/Night Owl-Light-color-theme-noitalic.json [hunk 1]
```diff
@@ -962,6 +962,21 @@
...
     },
+    {
+      "name": "CSS Custom Property declarations",
+      "scope": ["variable.css"],
+      "settings": {
+        "foreground": "#111111"
+      }
+    },
+    {
+				"name": "CSS Custom Property usage",
```
Comment (sdras) [inline themes/Night Owl-Light-color-theme-noitalic.json]: The formatting looks a little janky here, can you please fix it? Prettier can help with things like this.

**Comment by alexiscordova** [comment]
Since this is targeting variables in CSS files, will it not apply to custom properties in Sass/SCSS files? I made some tweaks locally but it doesn't seem like VSCode can differentiate between Sass variables and CSS Custom Properties. Is this correct? Thank you both!

File: themes/Night Owl-Light-color-theme-noitalic.json (hunk 1 above)
Comment (chiangs) [reply themes/Night Owl-Light-color-theme-noitalic.json]: weird...I use Prettier :) I will make the change now.

**Comment by chiangs** [comment]
@alexiscordova 

With my settings, you can see from the screen capture that it only applies in a CSS file and only selects the CSS Custom property.

Annotation 2019-12-10 103442

File: themes/Night Owl-Light-color-theme-noitalic.json (hunk 1 above)
Comment (chiangs) [reply themes/Night Owl-Light-color-theme-noitalic.json]: How odd, I use Prettier :)

I have updated the formatting in all themes.

**Comment by chiangs** [comment]
@sdras just a reminder that changes were made per your instructions :)

**Pull request closed without merging**